Examples/Movies/MovieCell.js

```diff
@@ -24,6 +24,7 @@ var {
   Text,
```

Comment:
Can we revert this file entirely? I don't see how any of these changes affect UWP.